src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
```

Comment:
nav should be in the separate file, it's BEM block so you have to create new file for navigation